src/rules/no-floating-promises-qawolf.ts

```diff
@@ -0,0 +1,13 @@
+import { reword } from "@denis-sokolov/eslint-plugin";
+import { builtinRules } from "eslint/use-at-your-own-risk";
+
+const baseRule = builtinRules.get("no-floating-promises");
+const rule = reword(
+  baseRule,
+  [
+    "Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator.",
+    "If you’re using `useEffect`, try some helper function like `useAsyncEffectAndCatch`.",
+  ].join(" "),
+);
+
+export default rule;
```

Comment:
The rules are not picked up automatically by virtue of creating a new file. You need to re-export it in src/rules/index.ts!

Reply:
[Remove "qawolf" suffix, and export](https://github.com/qawolf/eslint-plugin/pull/3/commits/efcd0c551aebffb088808e8d5f7845d6c5d98684)